web/classroom_views.py

```
@@ -0,0 +1,101 @@
+    try:
+        data = json.loads(request.body)
+        chair_id = data['chairId']
+        
+        # If user is sitting down
+        if data.get('sitting'):
+            # Remove user from any other chairs
+            ChairOccupancy.objects.filter(user=request.user).delete()
+            # Assign user to new chair
+            ChairOccupancy.objects.create(
+                chair_id=chair_id,
+                user=request.user
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)